contents/incentivization.rst

```diff
@@ -0,0 +1,189 @@
+
+The smart contract, after doing the appropriate sanity checks (checking the validity and the funds of the issuer), will transfer the funds from the issuer's contract to the beneficiary contract. If the cheque bounced due to insufficient funds in the issuer, the peer gets disconnected. Again, it is important to understand that a peer is able to freeride up to this threshold. The issuer cannot issue a cheque if there are insufficient funds; normal operation involves a smart contract call to check for funds. If a malicious node sends a bogus, malformed or altered cheque, the cheque will bounce and thus can be identified as fraud attempt, resulting in a disconnect.
+
+Refer to the documentation for details and specification of how the protocol handles subsequent cheques and how this evolves over time.
+
+Starting a SWAP enabled Swarm node thus requires a node to have funds. If no contract address is provided, the node will automatically create one, and a transaction is attempted when the node starts. The user can specify the amount to fund (in WEI) via command line parameter at boot (`swap-deposit-amount`). If the transaction failed, the node does not boot. Afterwards, the node remembers the contract address and uses it for future blockchain interaction.
+
+Currently, the node tries to cash in a cheque right away when it receives it via the protocol. In the future, Swarm might consider making this configurable by the user.
+
+To prevent fraudulent creation of contract addresses for the sake of stealing funds via the protocol, incentivized nodes need to contact an audited factory contract address when creating the new chequebook. For every blockchain, one factory should be deployed. For Ethereum networks, Swarm will provide the network address (and the deployment of the factory). The factory address needs to be configured via command line parameter for incentivized nodes (`swap-chequebook-factory`).
+
+.. important::
+  A public incentivized Swarm testnet is currently operational. For information on how to connect to it, please refer to chapter `6.8 <https://swarm-guide.readthedocs.io/en/latest/node_operator.html#connect-to-the-swap-enabled-testnet>`_. The testnet is experimental and funds can be lost at all times. Use only test tokens. The release of the public incentivized mainnet will be announced when launched.
+
+ 
+Smart contracts
+---------------
+The smart contract code is open source and there is a separate repo for it: `<https://github.com/ethersphere/swap-swear-and-swindle>`_.
```

Comment:
In an "official" documentation i would not recommend using abbreviations like repo.